schema/partners.js

```diff
@@ -36,8 +37,7 @@ const Partners = {
       `,
     },
     type: {
-      type: new GraphQLList(GraphQLString),
-      description: 'Only return partners of the specified _type(s).',
+      type: new GraphQLList(PartnerClassification.type),
```

Comment:
That's sort of my point I don't think it avoids the confusion. It's *fine* though.